docs/zh-cn/cli_commands.md

```diff
@@ -0,0 +1,442 @@
+```
+
+输出时间戳及VID:PID以替代键盘名：
+
+```
+qmk console -n -t
+```
+
+屏蔽bootloader的消息：
+
+```
+qmk console --no-bootloaders
+```
+
+## `qmk doctor`
+
+该命令用以检查你的开发环境并对发现的潜在的构建及刷写问题进行提醒，如果您乐意，它也可以修复其中大部分问题。
+
+**用法**：
+
+```
+qmk doctor [-y] [-n]
+```
+
+**示例**：
+
+检查开发环境中的问题并提示是否修复：
+
+    qmk doctor
+
+检查开发环境中的问题并自动进行修复：
+
+    qmk doctor -y
+
+检查开发环境中的问题，仅生成报告：
+
+    qmk doctor -n
+
+## `qmk format-json`
+
+将JSON文件格式化为（尽量）便于阅读的形式。会自动分辨JSON结构类型（info.json还是keymap.json），必要时也可以通过 `--format` 指定。
+
+**用法**：
+
+```
+qmk format-json [-f FORMAT] <json_file>
+```
+
+## `qmk info`
+
+展示QMK中的键盘及键映射信息，该命令用来获取键盘信息，输出配列，展示底层按键矩阵，及格式化地输出键映射JSON数据。
+
+**用法**：
+
+```
+qmk info [-f FORMAT] [-m] [-l] [-km KEYMAP] [-kb KEYBOARD]
+```
+
+该命令会尝试感知目录路径，当你在键盘或键映射目录下执行时，KEYBOARD及KEYMAP参数将被自动填入。
+
+**示例**：
+
+输出键盘的基础信息：
+
+    qmk info -kb planck/rev5
+
+输出键盘的矩阵信息：
+
+    qmk info -kb ergodox_ez -m
+
+输出键盘的键映射JSON数据：
+
+    qmk info -kb clueboard/california -km default
+
+## `qmk json2c`
+
+从QMK配置器导出的数据中生成 keymap.c 文件
+Creates a keymap.c from a QMK Configurator export.
+
+**用法**：
+
+```
+qmk json2c [-o OUTPUT] filename
+```
+
+## `qmk c2json`
+
+从 keymap.c 文件中生成 keymap.json
+**注意：** 解析C代码文件并不容易，该命令有可能无法对你的键映射文件生效，不使用C预处理代码有时可以解决问题。
+
+**用法**：
+
+```
+qmk c2json -km KEYMAP -kb KEYBOARD [-q] [--no-cpp] [-o OUTPUT] filename
+```
+
+## `qmk lint`
+
+检查键盘及键映射数据并提示出常见错误与问题，以及不符合模板规范的地方。
+
+**用法**：
+
+```
+qmk lint [-km KEYMAP] [-kb KEYBOARD] [--strict]
+```
+
+该命令会尝试感知目录路径，当你在键盘或键映射目录下执行时，KEYBOARD及KEYMAP参数将被自动填入。
+
+**示例**：
+
+基本的lint检查：
+
+    qmk lint -kb rominronin/katana60/rev2
+
+## `qmk list-keyboards`
+
+该命令可以列出 `qmk_firmware` 中所有的键盘
+
+**用法**：
+
+```
+qmk list-keyboards
+```
+
+## `qmk list-keymaps`
+
+该命令可以列出指定键盘（及指定版本）下的所有键映射。
+
+该命令会尝试感知目录路径，当你在键盘或键映射目录下执行时，KEYBOARD及KEYMAP参数将被自动填入。
+
+**用法**：
+
+```
+qmk list-keymaps -kb planck/ez
+```
+
+## `qmk new-keyboard`
+
+该命令可基于现有模板创建出新的键盘定义。
+
+对于未给出的参数，会提示你输入，若未传入 `-u` 参数且 .gitconfig 中设置了 `user.name`，则会提示你使用该值作为默认用户名。
+
+**用法**：
+
+```
+qmk new-keyboard [-kb KEYBOARD] [-t {avr,ps2avrgb}] -u USERNAME
+```
+
+## `qmk new-keymap`
+
+该命令可基于键盘已有的默认键映射创建新的键映射。
+
+该命令会尝试感知目录路径，当你在键盘或键映射目录下执行时，KEYBOARD及KEYMAP参数将被自动填入。
+
+**用法**：
+
+```
+qmk new-keymap [-kb KEYBOARD] [-km KEYMAP]
+```
+
+## `qmk clean`
+
+该命令会清理 `.build` 目录，若传入 `--all` 开关，在 `qmk_firmware` 下的所有.hex及.bin文件也会一并删除。
+
+**用法**：
+
+```
+qmk clean [-a]
+```
+
+---
+
+# 面向开发者的命令
+
+## `qmk format-text`
+
+该命令会重新格式化并统一文件的换行符。
+
+代码库下所有的文件须使用Unix换行符（LF）。
+若你在**Windows**下进行开发，必须确保文件中的换行符是正确的，才能让你的PR被允许合入。
+
+```
+qmk format-text
+```
+
+## `qmk format-c`
+
+该命令会使用clang-format来格式化C代码。
+
+不带参数地执行该命令以用来格式化核心代码相关的改动，默认会通过 `git diff` 来检查 `origin/master`， 可以通过 `-b <分支名>` 来改变检查的分支。
+
+带着 `-a` 开关执行命令会格式化所有的核心代码，也可以在命令行中传入文件名来指定格式化某个文件。
+
+**用以处理指定文件时**：
+
+```
+qmk format-c [file1] [file2] [...] [fileN]
+```
+
+**用以处理所有的核心代码时**：
+
+```
+qmk format-c -a
+```
+
+**用以处理 origin/master 下的所有改动时**：
+
+```
+qmk format-c
+```
+
+**用以处理指定分支下的所有改动时**：
+
+```
+qmk format-c -b branch_name
+```
+
```

Comment:
Missing https://github.com/qmk/qmk_firmware/blob/20beac5e571e3cb7694a53386c5b715d1bf6eee0/docs/cli_commands.md?plain=1#L385-L410

Reply:
added in latest commits